src/libpmemobj/obj.c

```diff
@@ -995,6 +995,17 @@ obj_descr_check(PMEMobjpool *pop, const char *layout, size_t poolsize)
 	return 0;
 }
 
+/*
+ * pmem_msync_nofail -- (internal) pmem_msync wrapper that never fails from
```

Comment:
I think it should be "obj_msync_nofail" instead of "pmem_msync_nofail"

Reply:
Done.